### PR TITLE
Remove date picker

### PR DESCRIPTION
**Pull request by gestchild**
References #2939

This removes the month picker from what's on and shows all the events together. @Heesoomoon are we happy for this to go in and then look at limiting the number displayed and adding a view more control etc. at a later date, with design input etc?

The same logic still applies to what we display, i.e. events which occur in multiple months, display once for each month in which they occur, showing and ordered by their next upcoming date for that month.

### CODE REVIEW

File: common/views/components/CardGrid/CardGrid.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ const CardGrid = ({
         <div className="css-grid">
           {items.map((item, i) => (
             <div
-              key={item.id}
+              key={i}
```
Comment (jamesgorrie) [inline common/views/components/CardGrid/CardGrid.js]: Why not the ID?

Comment (gestchild) [reply common/views/components/CardGrid/CardGrid.js]: because we duplicate events if they occur in more than one month

Comment (jamesgorrie) [reply common/views/components/CardGrid/CardGrid.js]: Could use `month-id` or something to make it more unique?

**Comment by gestchild** [comment]
Putting on hold for now

**Pull request closed without merging**